how-to-contribute.md

```diff
@@ -8,19 +8,20 @@ permalink: /contribute/
 ## Table of Contents
 1. [Introduction](how-to-contribute.md#introduction)
 2. [Install the repository](how-to-contribute.md#install-the-creatis-myriadgithubio-repository)
-3. [Setup a Ruby environment **(Linux)**](how-to-contribute.md#setup-a-ruby-environment-linux)
-4. [Setup a Ruby environment **(Windows)**](how-to-contribute.md#setup-a-ruby-environment-windows)
+3. [Setup a Ruby environment](how-to-contribute.md#setup-a-ruby-environment)
 5. [Add your own posts](how-to-contribute.md#add-your-own-posts)
 6. [Preview your posts](how-to-contribute.md#preview-your-posts-locally)
 7. [General troubleshooting](how-to-contribute.md#troubleshooting)
 
+&nbsp;
```

Comment:
I suspect because you edited this file in Windows, the end of line characters were somehow converted and saved in the file. Could you please revert them back?

Reply:
Ok. Just noticed these are rather HTML(?) tags to improve the spacing in the rendered page. You can disregard this comment :P